packages/z/zmqpb/xmake.lua

```diff
@@ -9,9 +9,15 @@ package("zmqpb")
     add_versions("0.2", "5dfa4d4cebb10cb7ae03943e18e8d48c8ff215e80371f24c5ade212be7f20721")
     add_versions("0.3", "343c57c9f72facca47082422a259ec8c531f5c6e332a3828835080c4a96b9064")
     add_versions("0.4", "7c0001db73b19e65b007adf6c9c5092c3589f043ab3e95a16b3ec2b2a87fa244")
+    add_versions("0.5", "d177c3b87e367932973da20b72376d1320c7501e0a2175c929d4c5e1f06f68ad")
 
     add_deps("cppzmq")
     add_deps("protobuf-cpp")
+    -- protobuf needs it and somehow just doesn't publicizes the linkage
+    add_deps( "abseil" )
+    add_deps( "utf8_range" )
+
+    add_configs("shared", {description = "Build shared binaries.", default = false, type = "boolean", readonly = false})
```

Comment:
so is still generated.

```
    "/home/runner/.xmake/packages/z/zmqpb/0.5/98c46d812b8c40388d90d102c2ec2e9b/lib/libZmqPb.so",
    "/home/runner/.xmake/packages/z/zmqpb/0.5/98c46d812b8c40388d90d102c2ec2e9b/lib/libZmqPb.so.0.5.0",
    "/home/runner/.xmake/packages/z/zmqpb/0.5/98c46d812b8c40388d90d102c2ec2e9b/lib/libZmqPb.so.0", 
  },
```

Reply:
zmqpb need to disable shared lib if run `xmake f -k static`

Reply:
i saw that in another package, but i took it out just now